plugins/auth-backend/src/identity/TokenFactory.ts

```diff
@@ -190,7 +189,7 @@ export class TokenFactory implements TokenIssuer {
         alg: key.alg,
         kid: key.kid,
       },
-      payload: { sub, ent, iat, exp },
```

Comment:
Did we want to remove `ent` from all tokens now that the user info service is flushed out or just from limited ones?

Reply:
The goal is to remove it from all tokens. But we may want to keep it in there still for some time, while the userInfo usage ramps up. As long as plugins exist that look at the claim directly (via an old IdentityService for example), the data may need to stay. It's fine to make the deprecation of that field a separate effort from implementing the endpoint fully.